api/src/main/java/com/epam/pipeline/acl/user/UserApiService.java

```diff
@@ -131,6 +131,11 @@ public PipelineUser updateUserRoles(Long id, List<Long> roles) {
         return new ArrayList<>(userManager.loadAllUsers());
     }
 
+    @PreAuthorize(ADMIN_OR_GENERAL_USER + OR_USER_READER)
```

Comment:
This method shall have the access as `loadUsers` - `ADMIN_ONLY + OR_USER_READER` (according to the task)